src/porepy/numerics/parabolic.py

```diff
@@ -64,30 +65,41 @@ def space_disc(self):
     problem.solve()
     '''
 
-    def __init__(self, gb, physics='transport', **kwargs):
+    def __init__(self, gb, physics='transport',time_step=1.0, end_time=1.0, **kwargs):
```

Comment:
I added time_step and end_time to the init.
This way the number of cases where you have to overload Parabolic is reduced drastically

Reply:
Fair enough.